src/control/server/mgmt_svc.go

```diff
@@ -312,10 +317,39 @@ func checkIsMSReplica(mi *IOServerInstance) error {
 	return nil
 }
 
+// validatePool ensures minimum SCM/NVMe pool size per storage target.
+func (svc *mgmtSvc) validatePool(req *mgmtpb.PoolCreateReq) error {
+	minScmBytes, err := humanize.ParseBytes(minScmSize)
```

Comment:
My instinct is to have the constants be the size in bytes, and only translate them into strings where necessary for display.